src/actions/arcActions.ts

```diff
@@ -127,21 +127,21 @@ export async function getDAOData(avatarAddress: string, getDetails: boolean = fa
     const getMintTokenEvents = promisify(mintTokenEvents.get.bind(mintTokenEvents));
     let eventsArray = await getMintTokenEvents();
     for (let cnt = 0; cnt < eventsArray.length; cnt++) {
-      memberAddresses.push(eventsArray[cnt].args.to);
+      memberAddresses.push(eventsArray[cnt].args.to.toLowerCase());
```

Comment:
Just wondering:  Would it make more sense for the member addresses to be accessors (via `get`) that convert the value, when requested, to lowercase?   Just wondering.  Maybe define a type that is an address that encapsulates the actual string (and maybe provides other functionality too)?

Reply:
hmm. you mean create a class for members? im not quite sure how that would work easily with redux state stuff, ill think about it.

Reply:
yeah unfortunately there is not a clean easy way to do this with Redux.